dotcom-rendering/src/web/components/DynamicSlowMPU.tsx

```diff
@@ -48,7 +55,12 @@ const Card33_ColumnOfThree33_Ad33 = ({
 	return (
 		<UL direction="row">
 			{card33.map((card) => (
-				<LI percentage="33.333%" padSides={true} key={card.url}>
+				<LI
+					percentage="33.333%"
+					padSides={true}
+					key={card.url}
+					stretch={true}
```

Comment:
I think that this `stretch={true}` might have snuck into the commit by accident?

Reply:
Ah yeah, removed another but missed this one! Will take it out.